### ISSUE TITLE
Taiga is not running

### ISSUE THREAD

**Issue by j3mu5** [edmundhighcock/hassio-repository#1]
I followed your very detailed instructions and was able to run all containers with passwords. However, when running taiga, there is an error displayed in the browser

![image](https://github.com/edmundhighcock/hassio-repository/assets/50756558/2af1b05f-0cfc-4a09-9b0d-b344fa9f7977)

```
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/uk/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/hy/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/ne/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/fr/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/be/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/en/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/ka/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/pt/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/hu/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/ko/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/mn/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/hr/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/es/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/sk/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/az/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/el/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/ja/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/es_AR/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/nl/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/id/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/ro/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/pt_BR/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/ar/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/da/LC_MESSAGES
processing file django.po in /home/taiga/taiga-back/.venv/lib/python3.8/site-packages/django/contrib/postgres/locale/tr/LC_MESSAGES

0 static files copied to '/home/taiga/taiga-back/static', 999 unmodified.
Tailing logs...
[2023-11-03 12:37:50 +0000] [1484] [INFO] Starting gunicorn 19.9.0
[2023-11-03 12:37:50 +0000] [1484] [INFO] Listening at: http://0.0.0.0:8003 (1484)
[2023-11-03 12:37:50 +0000] [1484] [INFO] Using worker: sync
/usr/lib/python3.8/os.py:1023: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used
  return io.open(fd, *args, **kwargs)
[2023-11-03 12:37:50 +0000] [1508] [INFO] Booting worker with pid: 1508
[2023-11-03 12:37:50 +0000] [1482] [INFO] Starting gunicorn 20.1.0
[2023-11-03 12:37:50 +0000] [1482] [INFO] Listening at: http://0.0.0.0:8001 (1482)
[2023-11-03 12:37:50 +0000] [1482] [INFO] Using worker: sync
[2023-11-03 12:37:50 +0000] [1511] [INFO] Booting worker with pid: 1511
[2023-11-03 12:37:50 +0000] [1512] [INFO] Booting worker with pid: 1512
[2023-11-03 12:37:50 +0000] [1525] [INFO] Booting worker with pid: 1525
[2023-11-03 12:37:50 +0000] [1526] [INFO] Booting worker with pid: 1526
[2023-11-03 12:37:50 +0000] [1527] [INFO] Booting worker with pid: 1527

> taigaio-events@6.5.0 start:production /home/taiga/taiga-events
> NODE_ENV=production node ./src/index.js

[2023-11-03 12:37:50 +0000] [1536] [INFO] Booting worker with pid: 1536
[2023-11-03 12:37:50 +0000] [1541] [INFO] Booting worker with pid: 1541
{"message":"WS server is started","level":"info","service":"taiga-events","timestamp":"2023-11-03 12:37:50"}
{"message":"Liveness / Readiness server was started in http://localhost:3023/healthz","level":"info","service":"taiga-events","timestamp":"2023-11-03 12:37:50"}
usage: manage.py dump_project [-h] [-d DIR] [-f [plain|gzip]] [--version]
                              [-v {0,1,2,3}] [--settings SETTINGS]
                              [--pythonpath PYTHONPATH] [--traceback]
                              [--no-color] [--force-color]
                              project_slugs [project_slugs ...]
manage.py dump_project: error: the following arguments are required: project_slugs
Backup complete, sleeping for 61 minutes.
 
 -------------- celery@9547a9e0-taiga v5.2.3 (dawn-chorus)
--- ***** ----- 
-- ******* ---- Linux-6.1.59-x86_64-with-glibc2.29 2023-11-03 12:37:51
- *** --- * --- 
- ** ---------- [config]
- ** ---------- .> app:         taiga:0x7f34114da580
- ** ---------- .> transport:   amqp://taiga:**@9547a9e0-rabbitmq:5672/taiga
- ** ---------- .> results:     disabled://
- *** --- * --- .> concurrency: 4 (prefork)
-- ******* ---- .> task events: OFF (enable -E to monitor tasks in this worker)
--- ***** ----- 
 -------------- [queues]
                .> tasks            exchange=tasks(topic) key=task.default
                

[tasks]
  . djmail.tasks.retry_send_messages
  . djmail.tasks.send_messages
  . taiga.auth.token_denylist.tasks.flush_expired_tokens
  . taiga.export_import.tasks.delete_project_dump
  . taiga.export_import.tasks.dump_project
  . taiga.export_import.tasks.load_project_dump
  . taiga.projects.contact.services.send_contact_email
  . taiga.projects.notifications.tasks.send_bulk_email
  . taiga.projects.services.projects.delete_project
  . taiga.projects.services.projects.delete_projects
  . taiga.projects.userstories.services._async_tasks_after_backlog_or_sprint_order_change
  . taiga.projects.userstories.services._async_tasks_after_kanban_order_change
  . taiga.telemetry.tasks.send_telemetry
  . taiga.timeline.service.push_to_timelines
  . taiga.webhooks.tasks.change_webhook
  . taiga.webhooks.tasks.create_webhook
  . taiga.webhooks.tasks.delete_webhook
  . taiga.webhooks.tasks.resend_webhook
  . taiga.webhooks.tasks.test_webhook

[2023-11-03 12:37:51,588: INFO/Beat] beat: Starting...
[2023-11-03 12:37:51,590: INFO/MainProcess] Connected to amqp://taiga:**@9547a9e0-rabbitmq:5672/taiga
[2023-11-03 12:37:51,593: INFO/MainProcess] mingle: searching for neighbors
[2023-11-03 12:37:52,622: INFO/MainProcess] mingle: all alone
[2023-11-03 12:39:51,657: INFO/Beat] Scheduler: Sending due task send-bulk-emails (taiga.projects.notifications.tasks.send_bulk_email)
[2023-11-03 12:39:51,669: INFO/MainProcess] Task taiga.projects.notifications.tasks.send_bulk_email[ccd62447-153a-4137-8e0a-015be72f873e] received
```

**Comment by rensjbeekhuizen** [comment]
Hi @j3mu5, i was configuring this yesterday, and i ran into the same issue so i was digging around. I couldn't get it to work on my pc. however, it does work when i go to the plugin over nabu casa. 
![taiga](https://github.com/edmundhighcock/hassio-repository/assets/58296996/1d0d6156-4a4c-4ae6-9428-fb60c6ffe2dd)
![taiga 2](https://github.com/edmundhighcock/hassio-repository/assets/58296996/31a3bd52-e895-4db7-9b74-250aa99572f9)

**Comment by edmundhighcock** [comment]
Sorry everyone for the slow reply (my company switched to an enterprise github account so I stopped getting notifications from this one). Yes, unfortunately my nginx skills are still basic and I couldn't get it to work simultaneously over the local and the nabucasa urls. Any suggestions welcome. In the meantime I will amend the documentation.

Sorry to have wasted your time @j3mu5 , but very glad to hear you got it working @rensjbeekhuizen! I have just switched to a raspberry pi so now I will need to get it running on ARM, but then maybe I can have another crack at this.
